app/admin.py

```diff
@@ -110,34 +110,41 @@ class DonorAdmin(admin.ModelAdmin):
                     'telephone_number',
                     'want_receipt',
                     'customer_ref',
-                    'verified')
+                    'verified',
+                    'item_count')
     list_filter = ['want_receipt', 'province']
     search_fields = ['id', 'donor_name', 'telephone_number', 'mobile_number',
-                     'address_line', 'city', 'province', 'postal_code', 'customer_ref', 'email', ]
+                     'address_line', 'city', 'province', 'postal_code', 'customer_ref', 'email']
 
-    def get_donor(self, obj):
-        return obj.id
-    get_donor.short_description = 'Donor ID'
+    def item_count(self, obj):
+        count_of_donor = sum([donation.item_set.count() for donation in obj.donation_set.all()])
```

Comment:
These aren't really `count_of_donor`. These are more like `count_per_donor`. Count of donor reminds of number of donors.